index.html

```diff
@@ -114,4 +121,249 @@ <h3>2015</h3>
         <div class="newmes-tonic">
           <button type="submit" class="go-newmes">See projects</button>
         </div>
+      </div>
+    </section>
+
+    <section class="works-card">
+      <img class="snapshort" src="./images/snap2.png" alt="snapshoot" />
+      <div class="left-block">
+        <div class="first-container">
+          <h1 class="newme-title">
+            Multi-Post <br />
+            Stories
+          </h1>
+          <div class="event2">
+            <div class="word">
+              <h3>CANOPY</h3>
+            </div>
+            <img src="./images/Counter.png" alt="counter count" />
+            <div class="role">
+              <h3>Back End Dev</h3>
+            </div>
+            <img src="./images/Counter.png" alt="counter image" class="dot" />
+            <div class="year">
+              <h3>2015</h3>
+            </div>
+          </div>
+        </div>
+        <p class="newme-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li class="language1">
+            <p>html</p>
+          </li>
+          <li class="language2">
+            <p>css</p>
+          </li>
+          <li class="language3">
+            <p>javascript</p>
+          </li>
+        </ul>
+        <div class="newmes">
+          <button type="submit" class="go-newmes">See projects</button>
```

Comment:
- [ ] All the buttons follow the button styling: Active, Hover, and Pressed but this is not the case for you kindly make sure your button follows the button styling provided in the Figma design

<img width="739" alt="Screen Shot 2022-10-27 at 23 18 32" src="https://user-images.githubusercontent.com/74501523/198400229-5b11f2b3-7806-483d-89fb-f17c7d1ed474.png">